src/components/modals/authentification/ModalAuthInitView.tsx

```diff
@@ -0,0 +1,123 @@
+
+        <button
+          type="button"
+          role="button"
+          className="h-11 w-full flex justify-center items-center border border-solid border-vintedTextGrisClair rounded mt-6 cursor-not-allowed"
+          disabled
+        >
+          <span className="pr-[6px]">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 48 48"
+              width="24px"
+              height="24px"
+            >
+              <path
+                fill="#039be5"
+                d="M24 5A19 19 0 1 0 24 43A19 19 0 1 0 24 5Z"
+              />
+              <path
+                fill="#fff"
+                d="M26.572,29.036h4.917l0.772-4.995h-5.69v-2.73c0-2.075,0.678-3.915,2.619-3.915h3.119v-4.359c-0.548-0.074-1.707-0.236-3.897-0.236c-4.573,0-7.254,2.415-7.254,7.917v3.323h-4.701v4.995h4.701v13.729C22.089,42.905,23.032,43,24,43c0.875,0,1.729-0.08,2.572-0.194V29.036z"
+              />
+            </svg>
+          </span>
+          <span className="text-vintedTextGrisClair">
+            Continuer avec Facebook
+          </span>
+        </button>
+
+        <button
+          type="button"
+          role="button"
+          className="flex justify-center items-center h-11 w-full border border-solid border-vintedTextGrisClair rounded mt-2 cursor-not-allowed"
+          disabled
+        >
+          <span className="pr-[6px]">
+            <svg
+              width="24px"
+              height="24px"
+              viewBox="-3 0 262 262"
+              preserveAspectRatio="xMidYMid"
+            >
+              <path
+                d="M255.878 133.451c0-10.734-.871-18.567-2.756-26.69H130.55v48.448h71.947c-1.45 12.04-9.283 30.172-26.69 42.356l-.244 1.622 38.755 30.023 2.685.268c24.659-22.774 38.875-56.282 38.875-96.027"
+                fill="#4285F4"
+              />
+              <path
+                d="M130.55 261.1c35.248 0 64.839-11.605 86.453-31.622l-41.196-31.913c-11.024 7.688-25.82 13.055-45.257 13.055-34.523 0-63.824-22.773-74.269-54.25l-1.531.13-40.298 31.187-.527 1.465C35.393 231.798 79.49 261.1 130.55 261.1"
+                fill="#34A853"
+              />
+              <path
+                d="M56.281 156.37c-2.756-8.123-4.351-16.827-4.351-25.82 0-8.994 1.595-17.697 4.206-25.82l-.073-1.73L15.26 71.312l-1.335.635C5.077 89.644 0 109.517 0 130.55s5.077 40.905 13.925 58.602l42.356-32.782"
+                fill="#FBBC05"
+              />
+              <path
+                d="M130.55 50.479c24.514 0 41.05 10.589 50.479 19.438l36.844-35.974C195.245 12.91 165.798 0 130.55 0 79.49 0 35.393 29.301 13.925 71.947l42.211 32.783c10.59-31.477 39.891-54.251 74.414-54.251"
+                fill="#EB4335"
+              />
+            </svg>
+          </span>
+          <span className="text-vintedTextGrisClair">
+            Continuer avec Google
+          </span>
+        </button>
+
+        <button
+          type="button"
+          role="button"
```

Comment:
Amélioration repérée avec Sonarlint : The element button has an implicit role of button. Defining this explicitly is redundant and should be avoided.sonarlint(typescript:S6822)

Reply:
corrigé, j'avais ajouté le role piur mes tests